pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -93,6 +95,7 @@ public GraphFullScan(IDocumentUIContainer documentUIContainer)
             spectrumBtn.Visible = false;
             filterBtn.Visible = false;
             lblScanId.Visible = false; // you might want to show the scan index for debugging
+            toolBar.Visible = false;

```

Comment:
Not sure about this change. Does it lead to a sudden appearance of the toolbar after the spectrum loads? That's a big layout change. Why is it necessary now?

Reply:
I thought that showing a toolbar with some active controls in an empty window could be confusing to the user. There is no functional need for this change. Reverted.